worker/runfmusite/testcase.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
```

Comment:
this file is added to the repository in runfmusite and addfmusite. factor it out to a common directory worker/common/.

Reply:
finished.